src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php

```diff
@@ -117,7 +131,7 @@ public function indexAction(Request $request, CategoryFilters $filters)
             'layoutHeaderToolbarBtn' => $this->getCategoryIndexToolbarButtons($request, $currentCategoryId),
             'currentCategoryView' => $categoryViewData,
             'deleteCategoriesForm' => $deleteCategoriesForm->createView(),
-            'isSingleShopContext' => $this->get('prestashop.adapter.shop.context')->isSingleShopContext(),
+            'isSingleShopContext' => $multistoreContextChecker->isSingleShopContext(),
```

Comment:
I think this interface is a bit overkill now, because we can use the new `ShopContext` for this kind of checks:
- `$isSingleShopContext = $this->getShopContext()->getShopConstraint()->getShopId() !== null`
- `$isGroupShopContext = $this->getShopContext()->getShopConstraint()->getShopGroupId() !== null`
- `$isAllShopsContext = $this->getShopContext()->getShopConstraint()->forAllShops()`

This way we can slowly stop relying on the `PrestaShop\PrestaShop\Adapter\Shop\Context` adapter service and one day remove it